static/src/stylesheets/module/identity/_switchboard.scss

```diff
@@ -0,0 +1,212 @@
+
+/* Switchboard
+   ========================================================================== */
+
+$local-gutter: $gs-gutter / 1.75;
+$checkbox-size: $gs-gutter / 1.25;
+
+.manage-account__switchboard ul,
+.manage-account__switchboard li {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.manage-account__switchboard {
+
+    ul {
+        grid-template-columns: repeat(auto-fit, minmax(gs-span(3), 1fr));
+        grid-auto-rows: 1fr;
+        display: grid;
+        padding-top: $local-gutter;
+        grid-row-gap: $local-gutter;
+        grid-column-gap: $local-gutter;
+        * {
```

Comment:
Can we avoid using the `*`? Will `li` work here?

Reply:
oh wow that should be '> *'. how do you feel about _that_ however? it's always going to be a li but i like using > * on flexboxes and grids as a general practice so any potential children get their layout set up

Reply:
True. But the way I understand it, in order to parse the `*` selector, the browser will have to check every element, then every element's parent to determine whether to apply the styles. By explicitly specifying the `li` tag selector, the browser only has to check the parents of `li` elements.

Since [the only valid child of a `ul` element is an `li` element](https://www.w3.org/TR/html5/grouping-content.html#the-ul-element), we don't have to worry about the case of the `ul` having children of various tags.

Reply:
@SiAdcock ugh, i had this idea that this was fixed in modern browsers, can't believe they still read them like this. is there any updated material on the whole css selector subject?

Reply:
I think browsers still work like that, at least on the first render. Subsequent renders (e.g. if you add a class to the `<body>` are more performant in modern browsers. 

The most important CSS optimisation you can perform is to reduce nesting. I know using `* { }` without any descendant selector is as performant as a tag selector. The problem comes when you add that descendant selector the browser has to scan the DOM tree twice.

Can't think of any definitive CSS selector perf article off-hand, however [this one](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations) has a few general rules (TL;DR: nesting is bad)

Reply:
as a serial nester that's good to keep in mind, thanks!